IDE/Espressif/ESP-IDF/examples/wolfssl_server/main/server-tls.c

```diff
@@ -42,7 +42,87 @@
     #include <wolfssl/wolfcrypt/mem_track.h>
 #endif
 
-const char *TAG = "tls_server";
+#if defined(WOLFSSL_ESPWROOM32SE)
+static const char* TAG = "tls_server";
```

Comment:
Please standardize on `static const char*` for either build type.

Reply:
Fixed. Thanks.